### PR TITLE
Document instructions for building SDK in non SGX Windows Machines

### PR DESCRIPTION
**Pull request by anakrish**
Signed-off-by: Anand Krishnamoorthi <anakrish@microsoft.com>

This PR depends on #3155

### CODE REVIEW

**Comment by anakrish** [comment]
@swamymsft Once this PR goes in, you'll be able to build OE SDK on a non SGX Windows Machine and run simulation mode :)

File: docs/GettingStartedDocs/Contributors/SimulatorGettingStartedWindows.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# Getting Started with Open Enclave in Simulation mode on Windows Machines Without SGX Support.
```
Comment (dthaler) [inline docs/GettingStartedDocs/Contributors/SimulatorGettingStartedWindows.md:1]:
```suggestion
# Getting Started with Open Enclave in Simulation Mode on Windows Machines Without SGX Support
```

File: docs/GettingStartedDocs/Contributors/SimulatorGettingStartedWindows.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+## Platform requirement
+
+Intel® X86-64bit architecture
+
+A version of Windows OS :
+- For server: Windows Server 2016 or 2019
+- For client: Windows 10 64-bit version 1709 or newer
+- To check your Windows version, run `winver` from the command line
+
+*Note:* The following instructions assume running `powershell` as adminstrator.
```
Comment (dthaler) [inline docs/GettingStartedDocs/Contributors/SimulatorGettingStartedWindows.md:12]:
```suggestion
*Note:* The following instructions assume running `powershell` as administrator.
```

File: docs/GettingStartedDocs/Contributors/SimulatorGettingStartedWindows.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+```powershell
+./scripts/install-windows-prereqs.ps1 -InstallPath C:/oe_prereqs -LaunchConfiguration SGX1-NoDriver -DCAPClientType None
+```
+
+## Building on Windows using Developer Command Prompt
+
+Launch the [x64 Native Tools Command Prompt for VS(2017 or 2019)](
+https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs),
+which is found in the `Visual Studio 2017` folder in the Start Menu.
```
Comment (dthaler) [inline docs/GettingStartedDocs/Contributors/SimulatorGettingStartedWindows.md:59]:
```suggestion
which is found in the `Visual Studio 2017` (or 2019) folder in the Start Menu.
```

File: docs/GettingStartedDocs/Contributors/SimulatorGettingStartedWindows.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+For more information refer to the [Advanced Test Info](AdvancedTestInfo.md) document.
+
+## Install
+
+The `ninja install` command will install the SDK in `C:/openenclave`
+
+## Build and run samples
+
+To build and run the samples, please look [here](/samples/README_Windows.md).
```
Comment (dthaler) [inline docs/GettingStartedDocs/Contributors/SimulatorGettingStartedWindows.md:147]:
```suggestion
To build and run the samples, see the [Windows samples README file](/samples/README_Windows.md).
```

**Comment by anakrish** [comment]
> @swamymsft Once this PR goes in, you'll be able to build OE SDK on a non SGX Windows Machine and run simulation mode :)

To try out do:

You need to do as describeed in the doc. But only diff is that instead of doing 
git clone --recursive https://github.com/openenclave/openenclave
 
You have to do:
git clone --recursive https://github.com/anakrish/openenclave
And then checkout the ak_delay_load branch via
git checkout ak_delay_load

**Comment by anakrish** [comment]
> Thanks @anakrish for writing this up. This will be really helpful to a lot of developers.
> May I request that you rename SimulatorGettingStartedWindows.md to WindowsSimulatorGettingStarted.md. This is just so naming is consistent in this folder.
> Additionally, can you provide a link to this file here: https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/Contributors/building_oe_sdk.md (3. Build, install and run).
> Additionally, we would need some documentation for end users of the SDK as seen here :https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Simulation.md for Linux
> 
> We can put the end user docs for using the SDK package in simulation mode on Windows in a different PR.

As suggested, I have added to link to the document in `building_oe_sdk.md`.

**Comment by anakrish** [comment]
> Please update the "Intel SGX" section of docs/GettingStartedDocs/README.md
> with a link to the new page.

@dthaler "Intel SGX" section of docs/GettingStartedDocs/README.md documents installing OE SDK. This document describes building OE SDK on non SGX Windows Machines.

**Comment by anakrish** [comment]
> > Please update the "Intel SGX" section of docs/GettingStartedDocs/README.md
> > with a link to the new page.
> 
> @dthaler "Intel SGX" section of docs/GettingStartedDocs/README.md documents installing OE SDK. This document describes building OE SDK on non SGX Windows Machines.

https://github.com/openenclave/openenclave/pull/3248 addresses the scenario of installing OE SDK for simulation mode on Windows.

**Comment by anakrish** [comment]
@dthaler Can you see if your blockers have been addressed?

**Comment by radhikaj** [comment]
Thanks @dthaler

**Pull request merged**